src/platform-apps/channels/styles.scss

```diff
@@ -45,6 +45,37 @@
       border-left: 3px solid #fb216e;
     }
   }
+
+  &-list__category-channel-name {
+    .Collapsible__trigger {
```

Comment:
Is it possible to have the css class name lowercase. I don't think we have any uppercase at this point?

Reply:
It looks like we can override it using `classParentString`: https://github.com/glennflanagan/react-collapsible#classparentstring--string--default-collapsible